docs-api/frontity-packages/features-packages/google-ad-manager.md

```diff
@@ -0,0 +1,205 @@
+            },
+          },
+        },
+      },
+    },
+  ],
+};
+```
+
+### Object properties
+
+| Name          | Type   | Required | Description                                                                                                                                                                              |
+| ------------- | ------ | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`slot`**    | string | yes      | The name of the slot as defined in your theme where you want the ad to go.                                                                                                               |
+| **`library`** | string | yes      | The React component used to display the Ad. We can set this value to `"googleAdManager.GooglePublisherTag"` (`GooglePublisherTag` component available in the `googleAdManager` package). |
+| `priority`    | int    | no       | Assigns a priority in case more than one fill is assigned to that slot.                                                                                                                  |
+| **`props`**   | obj    | yes      | Props that will be passed to the `<Slot>` component _(see table below)_                                                                                                                  |
+
+#### The `props` property
+
+An object with props that will be passed to the `<Slot>` component.
+
+| Name        | Type   | Required | Description                                                                                                                                                                                              |
+| ----------- | ------ | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`id`**    | string | yes      | An `id` for Ad block                                                                                                                                                                                     |
+| **`unit`**  | string | yes      | The (Google supplied) adUnitPath code for the ad unit to be displayed. _[more info](https://developers.google.com/publisher-tag/reference#googletag.slot-googletag.defineslotadunitpath,-size,-opt_div)_ |
+| **`size`**  | array  | yes      | The width and height to display the ad.                                                                                                                                                                  |
+| `targeting` | array  | no       | One or more keys, each with one or more associated values. _[more info](https://developers.google.com/publisher-tag/guides/key-value-targeting)_.                                                        |
```

Comment:
Instead of `array`, I would use `object` here.

BTW, you can take a look at the TSDocs of these properties: https://github.com/frontity/frontity/blob/3709480717760e1d9c716fdfc5dc01a016554b74/packages/google-ad-manager/types.ts#L23-L71